### DISCUSSION TITLE
Is the 'emitPackage' option available in yarn berry pnp?

### DISCUSSION THREAD

**Discussion by jungpaeng** [chakra-ui/panda#791] [Q&A] [answered]
I'm using yarn berry pnp to construct a monorepo.

And while following the desigSystem documentation, I ran into a problem: yarn berry pnp doesn't have access to node_modules.

Is it possible to use panda with yarn berry pnp?

**Comment by astahmer** [comment, marked as answer]
there's a little mention of this [here](https://panda-css.com/docs/guides/component-library#troubleshooting)

> If you use Yarn PnP, you might need to set the nodeLinker: node-modules in the .yarnrc.yml file.

tho, I understand this might not be ideal
I'm not a yarn pnp user so I don't really know what could work, do you have a solution in mind ? I think there is a `.yarn` folder, could we use it instead ?
